app/helpers/amendments_helper.rb

```diff
@@ -4,11 +4,11 @@ module AmendmentsHelper
   def collection_title(collection)
     return t('helpers.views.empty_collection', model: collection.model_name.human(count: 0).downcase) if collection.empty?
 
-    content_tag(:b, t('helpers.views.choose_model', model: collection.model_name.human.downcase)) + ':<br>'.html_safe
+    safe_join(content_tag(:b, t('helpers.views.choose_model', model: collection.model_name.human.downcase)), ':<br>')
   end
 
   def browse_title(object)
-    content_tag(:b, object.class.model_name.human) + ": #{object.full_title}" + '<br>'.html_safe
+    safe_join(content_tag(:b, object.class.model_name.human + ': '), content_tag(:span, object.full_title), '<br>')
```

Comment:
You should fix this `safe_join` call as the others.

Reply:
Fixed https://github.com/podemos-info/national_budgets/commit/8039afc8aa6e7e194df3b2d7823864dbced6a462